x-pack/plugins/security/public/management/users/components/confirm_delete_users/confirm_delete_users.tsx

```diff
@@ -99,6 +100,7 @@ export class ConfirmDeleteUsers extends Component<Props, unknown> {
           )
         );
       }
+
       if (callback) {
         callback(usersToDelete, errors);
       }
```

Comment:
Noticed that this callback is being executed for every individual delete, any idea if that's intentional or should this wait for all deletions to succeed or fail before calling the callback?

Reply:
Nice catch!

Just looked at the code, it's quite old but this is definitely an accident. The callback should only be executed once.

Would you mind adding a `.then()` and moving it there?

Reply:
Done